### PR TITLE
Improving the flow and grammar of docs/introduction/Motivation.md

### PR DESCRIPTION
**Pull request by markfinger**
These are general improvements for [docs/introduction/Motivation.md](docs/introduction/Motivation.md), mostly improving the grammar and flow.

### CODE REVIEW

File: docs/introduction/Motivation.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 
-As if this wasn’t bad enough, consider the **new requirements becoming common in front-end product development**, such as handling optimistic updates, rendering on the server, fetching data before performing route transitions, and so on. As front-end developers, we find ourselves surrounded by complexity we never had to deal with before, but [is it time to give up?](http://www.quirksmode.org/blog/archives/2015/07/stop_pushing_th.html)
+As if this wasn’t bad enough, consider the **new requirements becoming common in front-end product development**. As developers, we are expected to handle optimistic updates, server-side rendering, fetching data before performing route transitions, and so on. We find ourselves trying to manage a complexity that we have never had to deal with before, and we inevitably ask the question: [is it time to give up?](http://www.quirksmode.org/blog/archives/2015/07/stop_pushing_th.html) No.
```
Comment (omnidan) [inline docs/introduction/Motivation.md]: I like how you're answering the question but something like "The answer is *no*." probably sounds better.

**Comment by omnidan** [comment]
@markfinger thank you so much for this and #1011 - it does sound a lot better now! Once you've considered/fixed the small suggestion I have I'll merge this too :wink:

**Comment by markfinger** [comment]
Good catch, @omnidan. "No." is a bit blunt and aggressive.

I've applied your suggestion and it reads a lot better. Thanks for catching that!

**Comment by omnidan** [comment]
@markfinger thanks a lot - it does, yeah :grin:

**Pull request merged**